CONTRIBUTING.md

```diff
@@ -26,15 +26,33 @@ Install the [`aws-cli`](https://docs.aws.amazon.com/cli/latest/userguide/getting
 
 ## Executing Tests Locally
 
-Some of the tests require infrastructure components (databases etc.) to run locally. The easiest way to run all required components locally is to use Docker and on top of this [Docker Compose](https://docs.docker.com/compose/). Start the script `node bin/start-test-containers.js` to run all containers (not recommended).
-
-Instead:
+Some of the tests require infrastructure components (databases etc.) to run locally. The easiest way to run required components locally is to use Docker and on top of this [Docker Compose](https://docs.docker.com/compose/). 
 
 ```sh
 node bin/start-test-containers.js --mongo --redis
 ```
 
-It's not recommended to run all tests using `bin/run-tests.sh` - it takes too long. Take a look at the root package.json and run a specific test group or run e.g. `bin/run-collector-tests.sh` with the mocha `.only` attribute.
+### Using terminal aliases
+
+Add aliases to your terminal:
+
+```sh
+node bin/add-test-aliases.js bash|zsh
```

Comment:
`node bin/add-test-aliases.js bash` worked for me.

Reply:
I verified in zsh.

`node bin/add-test-aliases.js zsh` worked fine for me!